RELEASE.md

```diff
@@ -63,54 +63,18 @@ additional commits, see [Addressing issues with a Release](#addressing-issues-wi
 Once the release tag is pushed, a Buildkite build will be triggered on the release tag.
 
 You can view/adminster Buildkite builds [here](https://buildkite.com/chef/habitat-sh-habitat-master-release).
-
-The release tag builds will upload all release binaries to a channel named `rc-[VERSION]` and the `hab` cli will be uploaded but _not_ published to the `stable` Bintray repository. These builds can take about 45 minutes to fully complete. Keep an eye on them so we can validate the binaries when they finish.
-
-## Create a Release Notes Blog Post PR
-
-While you wait for the release automation to run, start working on the announcement.
-
-If you haven't posted to the blog before, see [this article](https://github.com/habitat-sh/habitat/tree/master/www#how-to-add-a-blog-post) since there may be some setup required. Also, don't forget to build and test your post locally before merging it.
-
-Create a "Release Notes" blog post file:
+When you get to the "Ensure that Builder is stable on the new release" step, you need to ssh to the various
+hosts and confirm that the new version of the supervisor is running and check the builder dashboards
+(see https://forums.habitat.sh/t/on-call-engineering-duties/626). To quickly check the versions of the supervisor,
+you can run (assuming you already set up the [ssh-helpers](https://github.com/habitat-sh/builder/blob/master/tools/ssh_helpers/Usage.md):
+```bash
+for host in $(./tools/ssh_helpers/hab-instances live | jq -r '.Reservations[] | .Instances[0] | .PublicDnsName + ";" + (.Tags | from_entries | ."X-Environment" + "-" + .Name)' | cut -d';' -f2); do
```

Comment:
Might want to make it clear that you're executing the `hab-instances` script from a `habitat-sh/builder` checkout, and not a `habitat-sh/habitat` one.